### ISSUE TITLE
Use a new KMS to create a DID by VDRI

### ISSUE THREAD

**Issue by kdimak** [hyperledger/aries-framework-go#2019]
Now the legacy one is used, what results in producing of Public Key which cannot be later found in the new KMS registry.

**Comment by llorllale** [comment]
Yeah - I've unfortunately had to use the legacykms as well as the new one when creating DIDs.
